samples/cpp/benchmark_app/main.cpp

```diff
@@ -699,17 +699,24 @@ int main(int argc, char* argv[]) {
         // ----------------- 8. Querying optimal runtime parameters
         // -----------------------------------------------------
         next_step();
+        slog::info << "Model: " << slog::endl;
```

Comment:
This outputs parameters that are not present in the core.get_rpoperty(all_properties) (requested in the ticket)